python/ray/rllib/agents/dqn/apex.py

```diff
@@ -23,7 +23,6 @@
         "learning_starts": 50000,
         "train_batch_size": 512,
         "sample_batch_size": 50,
-        "max_weight_sync_delay": 400,
```

Comment:
Good catch @michaelschaarschmidt; seems to be an accidental remove.

Reply:
@michaelschaarschmidt actually this was already provided in L15, so this is a duplicate 

(sorry for the previous message)

Reply:
oops, thanks!